Error while checking build/O2/fullCI for 77d0643cc585b59001f5c350ea1ebf269ee02af7 at 2023-11-30 20:47:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/O2Physics-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/QualityControl-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/O2-full-system-test-latest/log
Detected critical problem in logfile reco_ASYNC.log
[73804:internal-dpl-ccdb-backend]: [19:46:55][ERROR] CcdbDownloader finished transfer http://alice-ccdb.cern.ch/GLO/Config/SVertexerParam for 1550600800000 (agent_id: alissandra02-1701373601-EjSmcg) with http code: 404
[73804:internal-dpl-ccdb-backend]: [19:46:55][ERROR] File GLO/Config/SVertexerParam could not be retrieved. No more hosts to try.
[73812:ctp-entropy-decoder]: [19:46:55][ERROR] LM:202 L0:0 L1:1 TwI:3 Trigger classes wo input:166
[ERROR] pid 73853 (secondary-vertexing) crashed with or was killed with exit code 1
[ERROR]  - Device secondary-vertexing: pid 73853 (exit 1)
[ERROR] SEVERE: Device secondary-vertexing (73853) returned with 1


## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
++ GRERR=1
++ [[ 1 == 0 ]]
++ mkdir -p /sw/INSTALLROOT/d0fd8663a2a1df8ca41d21cd4389e7245bb576a9/slc8_x86-64/o2checkcode/1.0-local177/etc/modulefiles
++ cat
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/12332/77d0643cc585b59001f5c350ea1ebf269ee02af7/build_O2_fullCI/pretty.html).

--- a/Detectors/Vertexing/include/DetectorsVertexing/SVertexerParams.h
+++ b/Detectors/Vertexing/include/DetectorsVertexing/SVertexerParams.h
@@ -103,7 +103,7 @@ struct SVertexerParams : public o2::conf::ConfigurableParamHelper<SVertexerParam
   float minTPCdEdx = 250;                    // starting from this dEdx value, tracks with p > minMomTPCdEdx are always accepted
   float minMomTPCdEdx = 0.8;                 // minimum p for tracks with dEdx > mMinTPCdEdx to be accepted
   float maxV0TglAbsDiff = 0.3;               ///< max absolute difference in Tgl for V0 for photons only
-  float mTPCTrackMaxChi2 = 4.;               ///< max dca from prongs to vertex for photon TPC-only track only
+  float mTPCTrackMaxChi2 = 4.;               ///< max DCA from prongs to vertex for photon TPC-only track only
   float mTPCTrackMaxDZIni = 8.;              ///< don't consider as a seed (circles intersection) if Z distance exceeds this, for photon TPC-only track only
   float mTPCTrackMaxDXYIni = 8.;             ///< don't consider as a seed (circles intersection) if XY distance exceeds this, for photon TPC-only track only
   float mTPCTrackMaxDCAXY2ToMeanVertex = 2.; ///< max DCA^2 of V0 from beam line (mean vertex) for prompt V0 candidates, for photon TPC-only track only

